src/libraries/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -564,4 +567,7 @@
   <data name="net_http_chunk_too_large" xml:space="preserve">
     <value>The HTTP/1.1 response chunk was too large.</value>
   </data>
-</root>
\ No newline at end of file
+    <data name="net_unsupported_extended_connect" xml:space="preserve">
+    <value>Extended CONNECT is not supported.</value>
```

Comment:
Will folks know what an "Extended CONNECT" is, why it's happening, and what they can do about it?